src/docs/truffle/getting-started/using-the-truffle-dashboard.md

```diff
@@ -0,0 +1,104 @@
+---
+title: Using the Truffle Dashboard
+layout: docs.hbs
+---
+# Using the Truffle Dashboard
+
+When deploying your smart contracts you need to specify an Ethereum account that has enough funds to cover the transaction fees of the deployment. A popular method of doing this is copy-pasting your mnemonic phrase to a gitignored `.env` file so that it can be used for e.g. the `@truffle/hdwallet-provider`. But in general it is a bad practice to copy-paste your keys, especially since we have wallets like MetaMask that can send transactions for us.
+
+This is why we developed the Truffle Dashboard, an easy way to use your existing MetaMask wallet for your deployments and for other transactions that you need to send from a command line context. Because the Truffle Dashboard connects directly to MetaMask it is also possible to use it in combination with hardware wallets like Ledger or Trezor.
```

Comment:
```suggestion
We developed the Truffle Dashboard to provide an easy way to use your existing MetaMask wallet for your deployments and for other transactions that you need to send from a command line context. Because the Truffle Dashboard connects directly to MetaMask it is also possible to use it in combination with hardware wallets like Ledger or Trezor.
```

Reply:
Since github doesn't let me add a suggestion to a suggestion, please add a comma to:

"Because the Truffle Dashboard connects directly to MetaMask**,** it is also"